xclim/sdba/measures.py

```diff
@@ -442,3 +442,61 @@ def _scorr(
 scorr = StatisticalPropertyMeasure(
     identifier="Scorr", aspect="spatial", compute=_scorr, allowed_groups=["group"]
 )
+
+
+def _taylordiagram(
+    sim: xr.DataArray,
+    ref: xr.DataArray,
+    dim: str = "time",
+    group: str | Grouper = "time",
+) -> xr.DataArray:
+    """Taylor diagram.
+
+    Compute the standard deviation and Pearson correlation coefficient, necessary to plot points
+    on a Taylor diagram.
+
+    Parameters
+    ----------
+    sim : xr.DataArray
+        data from the simulation (a time-series for each grid-point)
+    ref : xr.DataArray
+        data from the reference (observations) (a time-series for each grid-point)
+    dim : str
+        Dimension across which the correlation and standard deviation should be computed.
+    group : str
+        Compute the property and measure for each temporal groups individually.
+        Currently not implemented.
+
+
+    Returns
+    -------
+    xr.DataArray, [measures]
+        DataAray containing the correlation coefficient and standard deviations as coordinates.
+    """
+    corr = xr.corr(sim, ref, dim=dim)
+
+    ref_std = ref.std(dim=dim, skipna=True, keep_attrs=True)
+    sim_std = sim.std(dim=dim, skipna=True, keep_attrs=True)
+
+    out = xr.DataArray(
+        [ref_std, sim_std, corr],
+        coords={"measures": ["ref_std", "sim_std", "corr"]},
+        dims=["measures"],
```

Comment:
this will break if you pass sim and ref that have more than just a time dimension. `StatisticalPropertyMeasure` should be able to have other extra dimensions (eg. lat and lon).